mock/runtime.go

```diff
@@ -1,43 +1,248 @@
 package mock
 
 import (
-	"context"
+	context "context"
```

Comment:
nit: Alias are not adding more readability

Reply:
updated by removing it